pkg/rpm_pfg.bzl

```diff
@@ -791,21 +794,30 @@ pkg_rpm = rule(
         # funny if it's not provided.  The contents of the RPM are believed to
         # be set as expected, though.
         "architecture": attr.string(
-            doc = """Package architecture.
+            doc = """Host architecture.
 
             This currently sets the `BuildArch` tag, which influences the output
             architecture of the package.
 
             Typically, `BuildArch` only needs to be set when the package is
-            known to be cross-platform (e.g. written in an interpreted
-            language), or, less common, when it is known that the application is
-            only valid for specific architectures.
+            not architecture dependent (e.g. written in an interpreted
+            language).
 
             When no attribute is provided, this will default to your host's
             architecture.  This is usually what you want.
 
             """,
         ),
+        "target_architecture": attr.string(
```

Comment:
Do we need both?  I have not figured out the value of setting both, since only one gets used.